diff -up OpenSubdiv-3_4_3/cmake/FindOpenCL.cmake.omv~ OpenSubdiv-3_4_3/cmake/FindOpenCL.cmake
--- OpenSubdiv-3_4_3/cmake/FindOpenCL.cmake.omv~	2021-11-20 11:17:20.770373112 +0100
+++ OpenSubdiv-3_4_3/cmake/FindOpenCL.cmake	2021-11-20 11:17:28.701393909 +0100
@@ -175,9 +175,9 @@ if(_OPENCL_CPP_INCLUDE_DIRS)
 
     list( REMOVE_DUPLICATES OPENCL_INCLUDE_DIRS )
     
-    if(EXISTS "${OPENCL_INCLUDE_DIRS}/CL/cl.h")
+    if(EXISTS "${OPENCL_INCLUDE_DIRS}/CL/cl_version.h")
     
-        file(STRINGS "${OPENCL_INCLUDE_DIRS}/CL/cl.h" LINES REGEX "^#define CL_VERSION_.*$")
+        file(STRINGS "${OPENCL_INCLUDE_DIRS}/CL/cl_version.h" LINES REGEX "^#define CL_VERSION_.*$")
 
         foreach(LINE ${LINES})
         
